knpu/authorization-code.md

```diff
@@ -124,12 +129,14 @@ a little further down::
         // equivalent to $_GET['code']
         $code = $request->get('code');
 
-        die('Implement this in CoopOAuthController::receiveAuthorizationCode');
+        die('Implement this in CoopOAuthController:receiveAuthorizationCode');
```

Comment:
@larzuk91 Gotcha! Auto-replace is dangerous thing, but PhpStorm allows you to do auto-replace with reviewing, which should prevent this cases ;)